pmd-java/etc/grammar/Java.jjt

```diff
@@ -3128,17 +3132,17 @@ void TryStatement() :
   [ FinallyStatement() ]
 }
 
-void ResourceSpecification() :
+void ResourceSpecification() #ResourceList:
```

Comment:
We should directly rename it instead of using the `#..` notation. The node is only references directly above in the TryStatement, so it should cause no problem...